packages/connectors/templates/preset/tsconfig.base.json

```diff
@@ -4,6 +4,7 @@
     "moduleResolution": "nodenext",
     "module": "esnext",
     "outDir": "lib",
-    "baseUrl": "."
+    "baseUrl": ".",
+    "jsx": "react-jsx"
```

Comment:
📝 Same here, should probably make distinction between clientside-enabled connector and otherwise